applications/callflow/src/module/cf_device.erl

```diff
@@ -55,7 +55,8 @@ bridge_to_endpoints(Data, Call) ->
     case cf_endpoint:build(EndpointId, Params, Call) of
         {'error', _}=E -> E;
         {'ok', Endpoints} ->
+            FailOnSingleReject = wh_json:get_value(<<"fail_on_single_reject">>, Data, 'undefined'),
             Timeout = wh_json:get_integer_value(<<"timeout">>, Data, ?DEFAULT_TIMEOUT_S),
             IgnoreEarlyMedia = cf_util:ignore_early_media(Endpoints),
-            whapps_call_command:b_bridge(Endpoints, Timeout, <<"simultaneous">>, IgnoreEarlyMedia, Call)
+            whapps_call_command:b_bridge(Endpoints, Timeout, <<"simultaneous">>, IgnoreEarlyMedia, 'undefined', 'undefined', 'undefined', FailOnSingleReject, Call)
```

Comment:
Hi @StevenGolemme,

First, thanks for the improvement. AFAICT this is a useful addition!

My only request is to build the bridge AMQP payload here now. The `whapps_call_command` functions were meant to be shortcuts to oft-used invocation. I think we've passed that point here.

Does that make sense?